src/Uno.UI/UI/Xaml/Media/ImageSource.crossruntime.cs

```diff
@@ -88,30 +88,7 @@ private protected void InvalidateSource()
 			_imageData = default;
 			if (_subscriptions.Count > 0 || this is SvgImageSource)
 			{
-				RequestOpen();
-			}
-		}
-
-		private protected void RequestOpen()
-		{
-			try
-			{
-				if (TryOpenSourceSync(null, null, out var img))
-				{
-					OnOpened(img);
-				}
-				else
-				{
-					_opening.Disposable = null;
-
-					_opening.Disposable = Uno.UI.Dispatching.NativeDispatcher.Main.EnqueueCancellableOperation(
-						ct => _ = Open(ct));
-				}
-			}
-			catch (Exception error)
-			{
-				this.Log().Error($"Error loading image: {error}");
-				OnOpened(ImageData.FromError(error));
+				_ = Open(CancellationToken.None);
```

Comment:
This is cancellable, since the outer async call can be cancelled by a changing property:

https://github.com/unoplatform/uno/blob/ebcdac6712e2852577537920810d74820eb4cb67/src/Uno.UI/UI/Xaml/Media/Imaging/BitmapSource.cs#L114-L116

This is particularly important when large lists of images are scrolled fast, and the datatemplates are rebound very quickly.